src/test/java/org/isf/admission/Tests.java

```diff
@@ -1244,6 +1493,102 @@ void testMgrValidateMaternity(boolean maternityRestartInJune) throws Exception {
 		admission.setAbortDate(abortDate);
 	}
 
+	@ParameterizedTest(name = "Test with MATERNITYRESTARTINJUNE={0}")
+	@MethodSource("maternityRestartInJune")
+	void testMgrGetAdmissionsPageable(boolean maternityRestartInJune) throws Exception {
+		GeneralData.MATERNITYRESTARTINJUNE = maternityRestartInJune;
+		// given:
+		int id = setupTestAdmission(false);
+		String str = "2000-01-01T10:11:30";
+		String str2 = "2023-05-05T10:11:30";
+		LocalDateTime dateFrom = LocalDateTime.parse(str);
+		LocalDateTime dateTo = LocalDateTime.parse(str2);
+		int page = 0;
+		int size = 10;
+		Admission foundAdmission = admissionIoOperation.getAdmission(id);
+		// when:
+		PagedResponse<Admission> patients = admissionBrowserManager.getAdmissionsPageable(dateFrom, dateTo, page, size);
+
+		// then:
+		assertThat(patients.getData().get(0).getId()).isEqualTo(foundAdmission.getId());
+	}
+
+	@ParameterizedTest(name = "Test with MATERNITYRESTARTINJUNE={0}")
+	@MethodSource("maternityRestartInJune")
+	void testMgrGetDischargesPageable(boolean maternityRestartInJune) throws Exception {
+		GeneralData.MATERNITYRESTARTINJUNE = maternityRestartInJune;
+		// given:
+		int id = setupTestAdmission(false);
+		String str = "2000-01-01T10:11:30";
+		String str2 = "2023-05-05T10:11:30";
+		LocalDateTime dateFrom = LocalDateTime.parse(str);
+		LocalDateTime dateTo = LocalDateTime.parse(str2);
+		int page = 0;
+		int size = 10;
+		Admission foundAdmission = admissionIoOperation.getAdmission(id);
+		// when:
+		PagedResponse<Admission> patients = admissionBrowserManager.getDischargesPageable(dateFrom, dateTo, page, size);
+
+		// then:
+		assertThat(patients.getData().get(0).getId()).isEqualTo(foundAdmission.getId());
+	}
+
+	@ParameterizedTest(name = "Test with MATERNITYRESTARTINJUNE={0}")
+	@MethodSource("maternityRestartInJune")
+	void testMgrGetDischarges(boolean maternityRestartInJune) throws Exception {
+		GeneralData.MATERNITYRESTARTINJUNE = maternityRestartInJune;
+		// given:
+		int id = setupTestAdmission(false);
+		String str = "2000-01-01T10:11:30";
+		String str2 = "2023-05-05T10:11:30";
+		LocalDateTime dateFrom = LocalDateTime.parse(str);
+		LocalDateTime dateTo = LocalDateTime.parse(str2);
+		int page = 0;
+		int size = 10;
+		Admission foundAdmission = admissionIoOperation.getAdmission(id);
+		// when:
+		List<Admission> patients = admissionBrowserManager.getDischarges(dateFrom, dateTo, page, size);
```

Comment:
strange that `getDischarges` requires page and size like the pageable version... 🤔

Reply:
It seems also not used in other components, maybe is an old version of the same method?

Reply:
`getDischarges()` is used in the API repo in `AdmissionController`.

See https://github.com/informatici/openhospital-api/blob/0dab980a096894145ebb682c3ea6e79cbca778bd/src/main/java/org/isf/admission/rest/AdmissionController.java#L270

Reply:
That endpoint is using the pageable version

https://github.com/informatici/openhospital-api/blob/0dab980a096894145ebb682c3ea6e79cbca778bd/src/main/java/org/isf/admission/rest/AdmissionController.java#L278

Reply:
my bad; changes committed